### ISSUE TITLE
Screen reader announcing controls name twice as ‘Search pages’ present on ‘Pages’ popup.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#142]
User Experience:

User who depend on screen reader might get confused and misguided if screen reader announces ‘twice’ controls name.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select Pages button.

5. Navigate to the various controls of the "Pages" window.

6. Enable screen reader and navigate to ‘Search box’.

7. Verify the issue.

 

Actual Result:

Screen reader announcing controls name twice as ‘Search pages’ present on ‘Pages’ popup.

 

Expected result:

Screen reader should not announce control name twice as ‘Search pages’ present on ‘Pages’ popup, screen reader should announce it once.

 

Note:

Similar issue exist throughout the application.

Similar issue exist with Chrome.

 

Reference Links

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
